cosmotech/data_update_quest/core/redis/dump_data.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) - 2025 - Cosmo Tech
+# This document and all information contained herein is the exclusive property -
+# including all intellectual property rights pertaining thereto - of Cosmo Tech.
+# Any use, reproduction, translation, broadcasting, transmission, distribution,
+# etc., to any person is prohibited unless it has been previously and
+# specifically authorized by written means by Cosmo Tech.
+
+import redis
+import json
+from pathlib import Path
+
+from cosmotech.orchestrator.utils.translate import T
+
+from cosmotech.data_update_quest_cli.utils.logger import LOGGER
+
+
+def configuration(host, port, password):
```

Comment:
The configuration function feels strange, you do 2 things in it :
- Create a working redis connection
- Download the list of domains in the redis instance

The list of domains is not really part of a "configuration" function
You probably could separate it in 2 :
- One function "get_redis_client" that will return the Redis object (that could have more effects later to configure the connection)
- One function "get_domains" that take the Redis object as a parameter and will return the domain